example_python/generate_parameter_module_example/admittance_parameters.py

```diff
@@ -0,0 +1,1275 @@
+# flake8: noqa
```

Comment:
Can you delete this autogenerated file from this PR?

Reply:
Removed. I don't know how it slipped in there. Thanks for the catch